extensions/binding/org.eclipse.smarthome.binding.astro/META-INF/MANIFEST.MF

```diff
@@ -4,16 +4,17 @@ Bundle-Name: Astro Binding
 Bundle-SymbolicName: org.eclipse.smarthome.binding.astro;singleton:=true
 Bundle-Vendor: Eclipse.org/SmartHome
 Bundle-Version: 0.9.0.qualifier
-Bundle-RequiredExecutionEnvironment: JavaSE-1.8
-Bundle-ClassPath: .
-Import-Package: com.google.common.collect,
+Bundle-Copyright: Eclipse Public License v1.0
+Import-Package: com.google.common.base,
```

Comment:
Please remove this library, its unnecessary if you only need it to instantiate a new HashMap.